lib/sacastats_web/templates/page/login.html.heex

```diff
@@ -2,4 +2,4 @@
     <div class="col-sm-6 col-xl-3 col-xs-12 mx-auto">
       <%= link "Login with Discord", to: "/login/new", class: "btn btn-primary" %>
     </div>
-</div>
+</div>
```

Comment:
```suggestion
</div>

```
Newline - I will look into some GitHub actions/custom commit flow to format automatically for us